libraries/node-core-library/src/FileSystem.ts

```diff
@@ -260,6 +277,11 @@ export interface IFileSystemCreateLinkOptions {
    * The new path for the new symlink link to be created.
    */
   newLinkPath: string;
+
+  /**
+   * Specifies what to do if the target object already exists. Defaults to `AlreadyExistsBehavior.Error`.
+   */
+  alreadyExistsBehavior?: AlreadyExistsBehavior;
```

Comment:
@D4N14L  I noticed today that this new option applies to lots of APIs like `createSymbolicLinkFolder()` and `createSymbolicLinkJunction()`, but it is only implemented for `createHardLink()`

Reply:
I can make a quick PR to update.